data/rulesets.ts

```diff
@@ -1357,4 +1357,15 @@ export const Rulesets: {[k: string]: FormatData} = {
 		desc: "Bans move combinations on Pok\u00e9mon that weren't legal in Japanese versions of Gen 1.",
 		// Implemented in mods/gen1jpn/rulesets.ts
 	},
+	noswitch: {
+		effectType: 'Rule',
+		name: 'No Switch',
+		desc: 'Players cannot switch their Pok&eacute;mon, except with an item, move, or Ability.',
+		onBegin() {
+			this.add('rule', 'No Switch : Players cannot switch their Pok&eacute;mon');
```

Comment:
```suggestion
			this.add('rule', 'No Switch: All Pok\u00e9mon are trapped');
```

Reply:
Let me guess, you're French? Is that the language that has the space before `:`? Anyway, in English, we don't.

Reply:
> Let me guess, you're French? Is that the language that has the space before `:`? Anyway, in English, we don't.

Yes! Absolutely. I didn’t know.